submissions/LuckyDnepr/Hooli-popup-task/styles-menu.css

```diff
@@ -0,0 +1,248 @@
+    height: 250px;
+    overflow-y:hidden;
+    align-self:center;
+    gap: 5px;
+    display: grid;
+    grid-template-rows: repeat(4, 3.5em);
+    grid-template-columns: 45% 45%;
+    justify-items: stretch;
+    justify-content: center;
+
+
+}
+
+.nav_menu::-webkit-scrollbar {
+    width: 0px;
+}
+
+.nav_menu_item {
+    cursor: pointer;
+    list-style-type: none;
+    border: 1px solid rgb(250, 128, 114);
+    box-shadow: 5px 5px 5px rgba(250, 128, 114, 0.5);
+    border-radius: 5px;
+    align-self: stretch;
+    margin: 2px 0;
+    background-image: linear-gradient(white 90%, salmon 100%);
+    transition: text-shadow linear 200ms;
+    
+    
+}
+
+.show_more_item {
+    display: none;
+}
+
+.nav_menu_item:hover {
+    text-shadow: 0px -3px 8px salmon;
+    background-image: linear-gradient(white 75%, salmon 100%);
+}
+
+.nav_item_marker {
+    opacity: 0;
+}
+
+.nav_menu_item_link:focus>.nav_menu_item_link_container>.nav_item_marker {
+    opacity: 100%;
+}
+
+.nav_menu_item_link:focus>.nav_menu_item_link_container>p {
+    font-weight: bold;
+}
+
+.nav_menu_item:active {
+    background-image: linear-gradient(white 40%, salmon 100%);
+}
+
+.nav_menu_item_link_container {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content:flex-start;
+}
+
+.menu_show_more {
+    display: block;
+    border: 1px solid grey;
+    background-color: rgba(250, 128, 114, 0.25);
+    text-align: center;
+    width: 90%;
+    margin: auto;
+}
+
+#menu_show_more_label {
+    margin-left: 20px;
+    margin-right: 20px;
+}
+
+#menu_show_more_input {
+    appearance: none;
+}
+
+#menu_show_more_input:focus + label > span {
+    font-weight: bold;
+}
+
+/*Change "Show more" to "Show less" start*/
+#menu_show_more_input:checked ~ .nav_menu {
+    overflow-y:scroll;
+}
+
+#menu_show_more_input:checked ~ .nav_menu > .show_more_item {
+    display: block;
+}
+
+.show_less_span {
+    display: none;
+}
+
+#menu_show_more_input:checked + label > .show_more_span{
+    display: none;
+}
+
+#menu_show_more_input:checked + label > .show_less_span{
+    display: block;
+}
+/*Change "Show more" to "Show less" end*/
+
+
+#nav_keeper_input:checked ~ .nav {
+    transform: translateX(0);
+  }
+  
+
+/*Animate Menu Button start*/
+#nav_keeper_input:checked ~ .nav_keeper_lines > .line1 {
+transform: rotate(45deg) translateY(-4px) translateX(4px);
+}
+
+#nav_keeper_input:checked ~ .nav_keeper_lines > .line2 {
+transform: scaleY(0);
+}
+
+#nav_keeper_input:checked ~ .nav_keeper_lines > .line3 {
+transform: rotate(-45deg) translateY(4px) translateX(4px);
+}
+/*Animate Menu Button end*/
+
+
+@media only screen and (max-width: 1200px){
+    .nav {
+        width: 40%;
+    }
+}
+
+@media only screen and (max-width: 1024px){
+    .nav {
+        width: 50%;
+    }
+}
+
+@media only screen and (max-width: 720px){
+    .nav {
+        width: 50%;
+    }
+}
+
+@media only screen and (max-width: 480px){
+    .nav {
+        width: 100%;
+    }
+}
```

Comment:
Please, also add [prettier](https://prettier.io/docs/en/index.html) to your ide. It will format spaces in code